test/jit/test_backends.py

```diff
@@ -685,3 +687,75 @@ def test_execution(self):
     @skipIfRocm
     def test_errors(self):
         self.error_module_compiler_test.test_errors()
+
+"""
+Unit Tests for Nnapi backend with delegate
+"""
+# This is needed for IS_WINDOWS or IS_MACOS to skip the tests.
+@unittest.skipIf(TEST_WITH_ROCM or IS_SANDCASTLE or IS_WINDOWS or IS_MACOS or IS_FBCODE,
+                 "Non-portable load_library call used in test")
+class NnapiBackendPReLUTest(JitTestCase):
+    """
+    Test lowering a simple PRelU module to Nnapi backend.
+    TODO: After Nnapi delegate is finished T91991928,
+    add tests for running the module (currently only lowers the model)
+    """
+    def setUp(self):
+        super().setUp()
+        # Load nnapi delegate library
+        torch_root = Path(__file__).resolve().parent.parent.parent
+        p = torch_root / 'build' / 'lib' / 'libnnapi_backend.so'
+        torch.ops.load_library(str(p))
+
+        # Nnapi set up
+        # Avoid saturation in fbgemm
+        torch.backends.quantized.engine = 'qnnpack'
+
+        libneuralnetworks_path = os.environ.get("LIBNEURALNETWORKS_PATH")
+        if libneuralnetworks_path:
+            ctypes.cdll.LoadLibrary(libneuralnetworks_path)
+            print("Will attempt to run NNAPI models.")
+            self.can_run_nnapi = True
+        else:
+            self.can_run_nnapi = False
+
+    def test_execution(self):
+        # Save default dtype
+        module = torch.nn.PReLU()
+        default_dtype = module.weight.dtype
+        # Change dtype to float32 (since float 64 is not supported)
+        torch.set_default_dtype(torch.float32)
+        module = torch.nn.PReLU()
+        args = torch.tensor([[1.0, -1.0, 2.0, -2.0]]).unsqueeze(-1).unsqueeze(-1)
+
+        # Trace and lower PreLu module
+        traced = torch.jit.trace(module, args)
+        compile_spec = {"forward": {"inputs": args}}
+        nnapi_model = torch._C._jit_to_backend("nnapi", traced, compile_spec)
+
+        # Change dtype back to the default
+        torch.set_default_dtype(default_dtype)
+
+# First skip is needed for IS_WINDOWS or IS_MACOS to skip the tests.
+# Second skip is because ASAN is currently causing an error.
+# It is still unclear how to resolve this.
+# See https://fb.workplace.com/groups/1144215345733672/permalink/2010505639104634/
```

Comment:
Maybe replace this with a task?
They're still opaque in OSS code but I think better than links to internal forums.